order-service/src/main/resources/docker-database.yml

```
@@ -22,7 +22,7 @@ spring:
     database-platform: org.hibernate.dialect.MySQL8Dialect
     hibernate:
       ddl-auto: update
-    properties:
-      hibernate:
-        format_sql: true
-        show_sql: true
\ No newline at end of file
+#    properties:
+#      hibernate:
+#        format_sql: true
+#        show_sql: true
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)